components/partials/swap/advanced-settings.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div>
+    <span class="block font-bold uppercase tracking-widest mb-4">
+      {{ $t('trade.swap.advanced_settings') }}
+    </span>
+    <div class="flex items-center justify-start mb-4">
+      <span class="text-gray-500 text-md">
+        {{ $t('trade.swap.slippage_tolerance') }}
+      </span>
+      <IconInfoTooltip
+        class="ml-1"
+        :tooltip="$t('trade.swap.slippage_tolerance_tooltip')"
+      />
+    </div>
+    <div class="grid grid-cols-7 sm:grid-cols-9 gap-2">
+      <SlippagePresetButton
+        v-for="(slippageTolerancePreset, index) in slippageTolerancePresets"
+        :key="`slippage-tolerance-preset-${index}`"
+        :label="slippageTolerancePreset.label"
+        :value="slippageTolerancePreset.value"
+        :active="
+          Number(slippageTolerancePreset.value) === Number(slippageTolerance)
+        "
+        :class="
+          index === slippageTolerancePresets.length - 1 && 'hidden sm:block'
```

Comment:
Let's not do these "React style conditionals". We should use the preferred Vue way - using the object syntax 

```ts
:class="{ 'hidden sm:block': index === slippageTolerancePresents.length - 1}"
```

Reply:
Ah yes sorry, force of habit 😃